### PR TITLE
[FEAT] Added omitRequestBodies and omitResponseBodies

### PR DESCRIPTION
**Pull request by dersvenhesse**
I added the new options omitRequestBodies and omitResponseBodies. Since there is an option to omitHeaders separately from skipSensitiveData and for bodies only the name-based options I felt this is a good addition to allow a lot of variety in hiding possible sensitive data, e.g. by only hiding headers and request bodies.

### CODE REVIEW

**Comment by DannyDainton** [comment]
Hey @dersvenhesse 👋🏻 

Thanks for taking a look and adding this into the reporter options, let me take a look and get it merged 🏆

File: lib/dashboard-template.hbs [hunk 1]
```diff
@@ -1009,7 +1014,9 @@ body.theme-light code {
                 {{/with}}
                 {{/isNotIn}}
                 {{/unless}}
+                {{/unless}}}
```
Comment (DannyDainton) [inline lib/dashboard-template.hbs:1017]: There's an extra `}` on the end and that would cause a failure when run

File: .gitignore [hunk 2]
```diff
@@ -2,6 +2,7 @@ node_modules
 
 *.log
 *.log.*
+.idea/
```
Comment (DannyDainton) [inline .gitignore:5]: Not really sure what this one is for 🤔

**Comment by DannyDainton** [comment]
For some reason, the pipeline didn't run for this PR but that would have shown you that there were some failures in there. 

You should also be able to run these locally using `npm run test`

File: lib/dashboard-template.hbs (hunk 1 above)
Comment (dersvenhesse) [reply lib/dashboard-template.hbs]: Oh, I oversaw that. It's fixed.

File: .gitignore (hunk 2 above)
Comment (dersvenhesse) [reply .gitignore:5]: I opened the project in my IDE (IntelliJ) which always creates this folder to store project-related settings. I can remove this if you don't feel the need. ;)

Comment (DannyDainton) [reply .gitignore]: You could just add the same thing in the other file?

**Comment by dersvenhesse** [comment]
> For some reason, the pipeline didn't run for this PR but that would have shown you that there were some failures in there.
> 
> You should also be able to run these locally using `npm run test`

```
newman-reporter-htmlextra: duration 3m 53.7s

        newman-reporter-htmlextra: ok!

**Comment by DannyDainton** [comment]
> > You should also be able to run these locally using `npm run test`
> 
> ```
> newman-reporter-htmlextra: duration 3m 53.7s
> 
>         newman-reporter-htmlextra: ok!
> ```

Nice one - That would have initially failed with the extra `}` in the template.

**Comment by DannyDainton** [comment]
Looks good! I'll get that merged in and sort out a new release so that everyone can benefit from the new functionality 😄

**Comment by dersvenhesse** [comment]
Wunderful, thanks for handling all that so quickly. 🙌

**Pull request merged**